os/kernel/log_dump/log_dump.c

```diff
@@ -357,6 +358,7 @@ int log_dump_save(char ch)
 		}
 		int msg_compress = false;
 
+		writesize = CONFIG_LOG_DUMP_CHUNK_SIZE; /* reset writesize before compression */
```

Comment:
can we change the writesize before compress_block is called instead of here?

Reply:
Done